src/universal/auth/pages/RegisterPage/RegisterPage.tsx

```diff
@@ -34,7 +34,7 @@ export default class RegisterPage extends Component<any, any> {
 
   public async handleSubmit(data: IUser) {
     if (data && data.email) {
```

Comment:
it's calling `await registerUserAction`, but it's not required, I was waiting for the response in the original version.